spec/requests/pages_spec.rb

```diff
@@ -122,6 +122,7 @@
     it "has proper headline" do
       get "/contact"
       expect(response.body).to include("Contact")
+      expect(response.body).to include("@#{SiteConfig.social_media_handles['twitter']}")
```

Comment:
Not relevant to the _copy_ change perse, but when I went to see if this page was tested already or not, I noticed that we never tested for dynamic value that exists on this page. It seemed prudent to have it in there (you'll notice we have it for all the other dynamic values on the _other_ pages, so I threw it in this PR for free 🤑)